docs/users-guide/08-dashboard-filters.md

```diff
@@ -4,84 +4,130 @@
+- **To set a default value**: click on the filter's **gears** icon to open the filter sidebar. Under the **Settings** tab, enter a value in the **Default value** input field.
+- **To remove a filter**: click on the filter's **gears** icon to open the filter sidebar. Under the **Settings** tab, click **Remove** in the sidebar. If you accidentally remove a filter, just click **Cancel** in the top-right to exit dashboard editing mode without saving your changes. 
 
 ### Using filters
 
-Once you’ve added a filter to your dashboard, just click on it to select a value and activate the filter. To stop filtering, just click the blue X. To change the filter, click anywhere else on it.
+Once you’ve added a filter to your dashboard, just click on the filter to select a value and activate the filter. To stop filtering, just click the blue X. 
 
 ![Using a filter](images/dashboard-filters/08-use-filter.png)
 
+You can also set up a dashboard question to [update a filter on click](interactive-dashboards.md#cross---filtering-a-dashboard).
+
 ### Choosing between a dropdown or autocomplete for your filter
 
-Picking selections for a filter with lots of options is easier than ever before. If the field you're using for a filter has more than 100 unique values, you'll now automatically see a search box with autocomplete suggestions.
+If the column you're using for a filter has more than 100 unique values, you'll now automatically see a search box with autocomplete suggestions:
 
 ![Autocomplete](images/dashboard-filters/autocomplete.png)
 
-Fields with fewer than 100 distinct values will have display a list of all the options.
+Columns with fewer than 100 distinct values will list all options:
 
 ![List](images/dashboard-filters/list.png)
 
 In both cases, you can pick one or multiple selections for your filter.
 
 ![Multi-select](images/dashboard-filters/multi-select.png)
 
-If Metabase somehow picked the wrong behavior for your field, admins can go to the [Data Model](../administration-guide/03-metadata-editing.md) section of the admin panel and click on the gear icon by the field in question to manually choose between a list, a search box, or just a plain input box.
+If Metabase somehow picked the wrong behavior for your field, admins can go to the [Data Model](../administration-guide/03-metadata-editing.md) section of the admin panel and click on the **gear** icon by the field in question to specify a list, search box, or plain input box.
 
 ![Search options](images/dashboard-filters/search-options.png)
 
+### Linking filters
+
+You can also **link filters** so that a child filter knows to limit its choices based on the activation of a parent filter.
+
+Say you have two filters, one to filter by state, the other to filter by city. You can link the city filter to the state filter so that when someone filter for California, the city filter will "know" to only show cities in California. In this case, state is the parent filter, and city is the child filter.
```

Comment:
```suggestion
Say you have two filters, one to filter by state, the other to filter by city. You can link the city filter to the state filter so that when someone filters by California, the city filter will "know" to only show cities in California. In this case, state is the parent filter, and city is the child filter.
```